File: Aula_Ricardo/Programação_orientada_a_objetos/python/Classes_objetos.py
class Carro: 
    def __init__(self, marca, modelo, ano):
        self.marca = marca
        self.modelo = modelo
        self.ano = ano

    def exibir_detalhes(self):
        print(f'Marca:{self.marca}, Modelo:{self.modelo}, Ano:{self.ano}')

carro1 = Carro('Toyota', 'Land cruiser', '2010')
carro1.exibir_detalhes()

File: Aula_Ricardo/Programação_orientada_a_objetos/go/Classes_objetos.go
package main

import "fmt"

type Carro struct {
	Marca  string
	Modelo string
	Ano    int
}

func (c Carro) ExibirDetalhes() {
	fmt.Printf("Marca: %s, Modelo: %s, Ano: %d\n", c.Marca, c.Modelo, c.Ano)
}

func main() {
	carro1 := Carro{Marca: "Toyota ", Modelo: "Land cruiser ", Ano: 2010}
	carro1.ExibirDetalhes()
}
